shared/types/oracle.ts

```diff
@@ -6,6 +6,35 @@ import type {
   projects,
   requestStates,
 } from "@shared/constants";
+import type { Address } from "wagmi";
+import {ParsedOOV1GraphEntity,ParsedOOV2GraphEntity,ParsedOOV3GraphEntity} from "./graphql";
+
+export type Request = {
+  chainId: ChainId;
+  oracleAddress: Address;
+  oracleType: OracleType;
+  id: string;
+  identifier: string;
+  ancillaryData: string;
+  time: string;
+  requester: Address;
+} & (Partial<ParsedOOV1GraphEntity> | Partial<ParsedOOV2GraphEntity>);
+
+export type Assertion = {
+  chainId: ChainId;
+  oracleAddress: Address;
+  oracleType: OracleType;
+  id:string,
+  identifier: string;
+  asserter: string;
+  claim: string;
+  assertionTimestamp: string;
+  assertionId: string;
+} & Partial<ParsedOOV3GraphEntity>;
```

Comment:
these new types are basically the same but ive tried to cureate data that we know will exist even in the least informed calls for data.  these assumptions may break in the future, but at least we can easily pull out required props.